package.json

```diff
@@ -17,7 +17,7 @@
     "test": "dotenv -- turbo run test",
     "test:e2e": "dotenv -- turbo run test:e2e",
     "db:generate": "dotenv -- turbo run db:generate",
-    "db:migrate": "dotenv -- yarn --cwd ./apps/platform/ prisma migrate dev",
+    "db:migrate": "dotenv -- yarn --cwd /platform/ prisma migrate dev",
```

Comment:
For those who can not develop with docker they need to make sure that yarn cwd argument is set to `--cwd ./apps/platform/`

Reply:
Good catch 💙. Let's create a different command for this in that case. Asking users to make those changes is not ideal.

`docker:db:migrate`

Reply:
I have pushed a fix for this. ✅